### DISCUSSION TITLE
上一个版本即`mpv_config-2024.03.04.7z`在发布页面被删除了，可以再提供一下吗？

### DISCUSSION THREAD

**Discussion by wathper** [dyphire/mpv-config#58] [General]
原文件在我电脑上被意外删除了，我是直接比对文件夹来查看版本变动的，我想将`mpv_config-2024.03.04.7z`的设置迁移到新版本需要`mpv_config-2024.03.04.7z`这不版本的原文件。

**Comment by dyphire** [comment]
没有保留原整合包文件，新发布的整合包的配置文件和 3.4 那个相比只有一个提交的更改，之所以删除是因为最新提交修了一个 fuzzydir.lua 脚本的严重影响文件加载体验的问题

**Comment by wathper** [comment]
好的，上个版本的那个脚本我移除了之后，响应速度确实快了不少。
